build/command.js

```diff
@@ -61,21 +59,30 @@ const argv = yargs( process.argv.slice( 2 ) )
 	.option( "factory", {
 		type: "boolean",
 		description:
-			"Build the factory bundle. " +
-			"By default, a UMD bundle is built."
+			"Build the factory bundle. By default, a UMD bundle is built."
 	} )
 	.option( "slim", {
 		alias: "s",
 		type: "boolean",
-		description: "Build a slim bundle, which excludes " +
-			slimExclude.join( ", " )
+		description:
+			"Build a slim bundle, which excludes " + slimExclude.join( ", " )
```

Comment:
The changes above were from running prettier over the file once, which I do sometimes to get line length right.